simulations_info.md

```diff
@@ -0,0 +1,24 @@
+# List of Genesis simulations
+
+Spatial resolution is particle gravitional softening; any scale less than ~3x the softening is not reliable. Halo structural properties are only reliable when > 200 particles are present in the halo. 
+
+| Simulation Type | Box side length (Mpc/h) |  Particle Number | Particle mass (M⊙/h) | Minimum redshift current (planned) | Spatial Resolution (kpc/h) | Snapshot cadence |
+| --------------- | ----------------------- | ---------------- | -------------------- | ---------------------------------- | -------------------------- | ---------------- |
+| DM-only         | 35                      | 2650^3           | 2.0e5                |    0.0  (5.0)                      | ~0.25 
```

Comment:
@manodeep @pelahi should the Minimum redshifts for the DM-only super zoom be: current=5 and planned=0 ?

Reply:
Should be 5.0 (5.0) for this simulation.